### DISCUSSION TITLE
Parsing Json responses in Flows

### DISCUSSION THREAD

**Discussion by diferdin** [postmanlabs/postman-flows#387] [Q&A] [answered]
Re-posting this much-discussed issue here, as I haven't seen satisfactory answers.

I am building a flow that requires parsing of JSON response, and creating durables out of some fields in that JSON. Problem is, Flows will consider response body as one string, and I won't be able to access the single fields. Setting Parse Body to JSON won't get me there either, and using the call's Tests section to create global/environment/collection variables out of those fields' value won't work either (apparently it's not possible create or update variables through Tests when using Flows).

Any suggestion as to how to parse the body as JSON, or how to access a field inside a non-parsed JSON response body would be welcome.

Thank you in advance.

**Comment by Ariemeth** [comment, marked as answer]
Have you created examples for the request you are using?  Flows use the examples to know how to parse a response from a request.
